docs/source/intro/vocabulary.rst

```diff
@@ -25,26 +35,66 @@ The ASReview project makes use of standardized terminology for all communication
 +------------------+-----------------------------------------------------------+
 | **Extension**    | Use to indicate additional elements to the ASReview       |
 |                  | software, such as the ASReview visualisation extension,   |
 |                  | or the ASReview CORD-19 extension.                        |
 |                  |                                                           |
 +------------------+-----------------------------------------------------------+
-| **ELAS**         | Our Electronic Learning ASsistent. Name of our mascot.    |
-|                  | Use for storytelling and to increase explainability.      |
+| **Import**       | Use to describe the action of importing a dataset or a    |
+|                  | project into ASReview LAB.                                |
+|                  |                                                           |
+| **Export**       | Use to describe the action of exporting a dataset or a    |
+|                  | project from ASReview LAB.                                |
 |                  |                                                           |
 +------------------+-----------------------------------------------------------+
-| **User**         | The human annotator who labels records.                   |
+| **Mode**         | Use to refer to one of the three project modes in         |
+|                  | ASReview LAB: oracle, simulation, and exploration.        |
+|                  |                                                           |
+|                  | The **Oracle** mode is used when a user performs a        |
+|                  | systematic review in interaction with the AI, where the   |
+|                  | record tobe screened is the result of the model.          |
 |                  |                                                           |
-| **Screener**     | Replacement term when context is PRISMA-based reviewing.  |
+|                  | The **Exploration** mode includes several preloaded       |
+|                  | labelled datasets can be used for teaching purposes,      |
+|                  | benchmark testing and debugging.                          |
+|                  |                                                           |
+|                  | The **Simulation** mode is used for running simulations to|
+|                  | evaluate the performance of one (or multiple) active      |
+|                  | learning models on labelled datasets.                     |
+|                  |                                                           |
++------------------+-----------------------------------------------------------+
+| **Notes**        | Information added by the user in the note field and       |
+|                  | stored in the project file.                               |
+|                  | It can be changed in the history panel.                   |
```

Comment:
```suggestion
| **Notes**        | Information added by the user in the note field and       |
|                  | stored in the project file. It can be edited on the       |
|                  | project page **History**.                                 |
```